doc/source/whatsnew/v0.18.2.txt

```diff
@@ -51,6 +51,43 @@ New behaviour:
 
    In [2]: pd.read_csv(StringIO(data), names=names)
 
+.. _whatsnew_0182.enhancements.semi_month_offsets:
+
+Semi-Month Offsets
+^^^^^^^^^^^^^^^^^^
+
+The ``SemiMonthEnd`` ('SM') ``SemiMonthBegin`` ('SMS') offsets provide date offsets
```

Comment:
Pandas has gained new frequency offsets, ``SemiMonthEnd`` ('SM') and ``SemiMonthBegin`` ('SMS'). These provide date offsets ....